### PR TITLE
Mano algolia autocomplete  Fixes#4153

### PR DESCRIPTION
**Pull request by xmano**
Fix algolia autocomplete #4153

### CODE REVIEW

File: libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts [hunk 1]
```diff
@@ -140,15 +138,12 @@ export class AlgoliaAutocompleteComponent implements OnInit, OnDestroy {
...
    */
-  public displayFn() {
-    if (this.resetInput) {
-      return ''
-    }
+  public getSelectedOrg(orgName: string) {
     const value = this.lastValue$.getValue();
     if (value) {
-      return getDeepValue(value[0], this.keyToDisplay);
+      const orgSelected = value.filter(org => org[this.keyToDisplay] === orgName);
```
Comment (fritzschoff) [inline libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts:144]: same here for the var name

File: libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts [hunk 2]
```diff
@@ -140,15 +138,12 @@ export class AlgoliaAutocompleteComponent implements OnInit, OnDestroy {
    * This function is used internally by the MatAutocomplete to decide
    * what to display in the input field when an option has been selected
    */
-  public displayFn() {
```
Comment (fritzschoff) [inline libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts:143]: why do you deleted that function? Also if the developer sets `this.resetInput` to true, you won't reset the input

File: libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts [hunk 3]
```diff
@@ -140,15 +138,12 @@ export class AlgoliaAutocompleteComponent implements OnInit, OnDestroy {
...
-    if (this.resetInput) {
-      return ''
-    }
+  public getSelectedOrg(orgName: string) {
```
[the remark this answers is not in the document]
Comment (fritzschoff) [reply libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts:141]: Same for the parameter

**Review by fritzschoff — CHANGES_REQUESTED** [review]

File: libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (xmano) [reply libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts:141]: ok.. will change the names.

File: libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts (hunk 1 above)
Comment (xmano) [reply libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts:144]: ok.. will change the var name

File: libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts (hunk 2 above)
Comment (xmano) [reply libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts:143]: That function is not needed. But I need to check where to handle `resetInput`. Will update.

Comment (xmano) [reply libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts:143]: `
  public selected(result: any) {
    this.selectionChange.emit(result);
    if (this.resetInput) {
      this.control.reset();
    }
  }
`

wont the above code handle it?

Comment (xmano) [reply libs/ui/src/lib/algolia/autocomplete/algolia-autocomplete.component.ts:143]: function removed entirely.

**Comment by fritzschoff** [comment]
looks good now

**Pull request merged**